### DISCUSSION TITLE
Most recent build just broke OpenRouter connectivity

### DISCUSSION THREAD

**Discussion by bpawnzZ** [danny-avila/LibreChat#3406] [Troubleshooting]
### What happened?

I just rebuilt and deployed the latest pull of the code and 'OpenRouter' is now not showing up in drop down window.  went to https://www.librechat.ai/docs/configuration/librechat_yaml/ai_endpoints/openrouter and updated my Librechat.yml entry and still a no go.  Then rolled back to previous build and its working again.  Seems somethinng somewhere got broken.  Is there any update for Librechat.yml entry??  Just wondering and thanks.

### Steps to Reproduce

1. Do a git pull of latest codebase from github..
2. compile
3. openrouter connectivity/drop down borken

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Check your startup logs. There was a breaking change for speech config. If openrouter is no longer showing up, you likely have a config issue.

![image](https://github.com/user-attachments/assets/77b8778a-c7a9-484a-8831-97a18d7030c4)

**Reply by bpawnzZ** [reply]
Got you.. I will certainly check this out and i will let you know if I cant sort it out..  Thanks for the response.
